compiler/stmt.py

```diff
@@ -364,6 +364,12 @@ def visit_Import(self, node):
         self.block.bind_var(self.writer, asname, mod.expr)
 
   def visit_ImportFrom(self, node):
+    # Check for wildcard member import and raise error.
```

Comment:
According to the [Google Python style guide](https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments), comments should describe the purpose of the code, not what it actually does so maybe: `# Wildcard imports are not yet supported.`